Added sticky position term to CSS concepts (#5519)
Codecademy/docs: content/css/concepts/position/terms/sticky/sticky.md

---
Title: 'Sticky'
Description: 'Defines an element that toggles between relative and fixed positions depending on the scroll position.'
Subjects:
  - 'Web Design'
  - 'Web Development'
Tags:
  - 'CSS '
  - 'Positioning'
CatalogContent:
  - 'learn-css'
  - 'paths/front-end-engineer-career-path'
---

Sticky Position in CSS allows an element to toggle between relative and fixed positioning based on the person's scroll position. It commonly keeps certain elements, such as headers or navigation menus, in view when scrolling through content.

## Syntax

```pseudo

> position: sticky;

Reviewer: ```suggestion
position: sticky;
```

```

---
Title: 'Sticky'
Description: 'Defines an element that toggles between relative and fixed positions depending on the scroll position.'
Subjects:

>   - 'Web Development'
>   - 'Web Design'

Reviewer: ```suggestion
  - 'Web Design'
  - 'Web Development'
```

Tags:

>   - 'Positioning'
>   - 'CSS'

Reviewer: ```suggestion
  - 'CSS '
  - 'Positioning'
```

CatalogContent:

>   - 'learn-css'
>   - 'paths/front-end-engineer-career-path'
>   - 'paths/full-stack-engineer-career-path'

Reviewer: ```suggestion
  - 'learn-css'
  - 'paths/front-end-engineer-career-path'
```

---

> Sticky Position in CSS allows an element to toggle between relative and fixed positioning based on the user's scroll position. It's commonly used to keep certain elements, such as headers or navigation menus, in view when scrolling through content.

Reviewer: Rewrite according to Codecademy style
```suggestion
Sticky Position in CSS allows an element to toggle between relative and fixed positioning based on the person's scroll position. It commonly keeps certain elements, such as headers or navigation menus, in view when scrolling through content.
```

## Syntax

> ```css

Reviewer: ```suggestion
```pseudo
```

position: sticky;

> ```

Reviewer: ```suggestion
```

```

The `position: sticky` property is often used with length or percentage values applied to `top`,` right`, `bottom`, or `left` to define the threshold for switching between relative and fixed positioning.
    

> > **Note:**  To ensure `position:sticky` works, the containing block must have a defined height, and the sticky element should not be covered by other positioned elements.

Reviewer: ```suggestion
> **Note:**  To ensure `position: sticky` works, the containing block must have a defined height, and the sticky element should not be covered by other positioned elements.
```

> ## Example 1

Reviewer 2: Since there is only one example, we can simply use the heading `Example`.

```suggestion
## Example
```

> Make the `.header` element sticky, so it remains visible at the top when scrolling past it.

Reviewer: ```suggestion
Make the `.header` element sticky so it remains visible at the top when scrolling past it.
```
